### PR TITLE
rm unnecessary parameters for keystone prepare

### PR DESCRIPTION
**Pull request by dcousens**

### CODE REVIEW

File: test-projects/access-control/CHANGELOG.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
       ```diff
       +const express = require('express');
        const port = 3000;
-       keystone.prepare({ port })
```
Comment (dcousens) [inline test-projects/access-control/CHANGELOG.md]: Is the CHANGELOG changes necessary?
I haven't done it for every change log [yet]. There were quite a few.

Comment (jesstelford) [reply test-projects/access-control/CHANGELOG.md]: No, the `CHANGELOG`s are historical records and shouldn't be edited.

File: packages/keystone/lib/Keystone/index.js [hunk 2]
```diff
@@ -419,7 +419,7 @@ module.exports = class Keystone {
     return mergeRelationships(createdItems, createdRelationships);
   }
 
-  async prepare({ port = DEFAULT_PORT, dev = false, apps = [], distDir } = {}) {
+  async prepare({ dev = false, apps = [], distDir } = {}) {
```
Comment (jesstelford) [inline packages/keystone/lib/Keystone/index.js]: Can also remove the `DEFAULT_PORT` import from the top of this file.

File: test-projects/access-control/CHANGELOG.md (hunk 1 above)
Comment (dcousens) [reply test-projects/access-control/CHANGELOG.md]: Removed

File: packages/keystone/lib/Keystone/index.js (hunk 2 above)
Comment (dcousens) [reply packages/keystone/lib/Keystone/index.js]: Done

**Pull request merged**